src/poliastro/contrib/extract_czml.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import json
+
+from astropy.coordinates import CartesianRepresentation
+from astropy import units as u
+
+from poliastro.contrib.czml_extract_default_params import PIC_SATELLITE, DEFAULTS
+
+
+class ExtractorCZML:
```

Comment:
What about `CZMLExtractor` instead?

Reply:
It was initially ``CZMLExtractor``, but I though the camel case made it more readable that way. However, I have no problem to change it back if you think it'd be better

Reply:
Yeah, let's change it to `CZMLExtractor`, the other name feels a bit backwards